tools/lint.mjs

```diff
@@ -273,4 +273,9 @@ for (const match of source.matchAll(/^ *\d+\. .*$/mg)) {
   }
 }
 
+// Avoid incorrect links to list/empty.
+for (const match of source.matchAll(/is( not)? \[=(list\/|stack\/|queue\/|)empty=\]/g)) {
+  error(`Link to 'is empty' (noun) not 'empty' (verb): ${format(match)}`);
```

Comment:
Should 'is empty' be an (adjective)? Apologizes if I misunderstood as I am not a native speaker.

Reply:
Yes, it should be an adjective here.

(supposedly "empty" *can* be used as a noun, calling a "bottle" an "empty", but I've never heard it used this way in my life)

Reply:
Uh, yeah...  I'll fix the PR next week unless someone beats me to it.

Reply:
Total aside: it's common in the restaurant/bar world to refer to "empties", e.g. when serving tables ("Can I clear your empties?"), at least in my part of the world.